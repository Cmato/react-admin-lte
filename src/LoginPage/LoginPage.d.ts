import * as React from 'react';

export interface LoginPageProps {
}

declare const LoginPage: React.ComponentType<LoginPageProps>;

export default LoginPage;
