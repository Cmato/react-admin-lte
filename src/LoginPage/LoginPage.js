import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';

const LoginPage = ({
  children,
  className: classNameProp,
  ...other
}) => {
  const className = classNames({
    'login-page': true
  }, classNameProp);

  return (
    <div className={className} style={{ minHeight: '100vh' }} {...other}>
      {children}
    </div>
  );
};

LoginPage.propTypes = {
  /**
   * The contents of the component.
   */
  children: PropTypes.node.isRequired,
  /**
   * @ignore
   */
  className: PropTypes.string
};

export default LoginPage;
