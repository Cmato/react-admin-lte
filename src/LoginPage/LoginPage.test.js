import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from './LoginPage';

it('renders correctly', () => {
  expect(shallow(<LoginPage><div /></LoginPage>))
    .toMatchSnapshot();
});
