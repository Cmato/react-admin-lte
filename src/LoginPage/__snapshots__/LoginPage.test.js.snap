// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="login-page"
  style={
    Object {
      "minHeight": "100vh",
    }
  }
>
  <div />
</div>
`;
